### PR TITLE
for resource_id only - redirect to full url

### PR DESCRIPTION
**Pull request by nimitsarup**
/interactives/resid , redirect to /interactives/slug-resid/embed

### CODE REVIEW

File: handlers/handlers.go [hunk 1]
```diff
@@ -103,6 +87,46 @@ func streamFromStorageProvider(w http.ResponseWriter, r *http.Request, clients r
 	}
 }
 
+func canGetInteractive(w http.ResponseWriter, r *http.Request, id string, clients routes.Clients, serviceAuthToken string) *interactives.Interactive {
```
Comment (mryan321) [inline handlers/handlers.go]: minor suggestion - `getInteractive` is better name imo

**Review by mryan321 — DISMISSED** [review]
1 minor comment :)

File: routes/routes.go [hunk 2]
```diff
@@ -35,13 +35,14 @@ func Setup(_ *config.Config, r *mux.Router, hc http.HandlerFunc, interactivesHan
...
 		Handler(interactivesHandler)
+	// only resource_id - redirect
 	r.StrictSlash(true).
 		PathPrefix(getPath(false, false)).
 		MatcherFunc(func(r *http.Request, rm *mux.RouteMatch) bool {
 			return !strings.HasSuffix(r.URL.Path, EmbeddedSuffix)
 		}).
 		Methods(http.MethodGet).
-		Handler(interactivesHandler)
+		Handler(redirectHandler)
```
Comment (mryan321) [inline routes/routes.go:46]: sorry nimit - missed this - i think you wanna redirect the last route too - the embed one with just resource_id 👍

File: handlers/handlers.go (hunk 1 above)
Comment (nimitsarup) [reply handlers/handlers.go]: yes.. i had the function returning a bool .. hence canXX ..

**Review by mryan321 — APPROVED** [review]

**Pull request merged**